great_expectations/cli/toolkit.py

```diff
@@ -119,7 +127,7 @@ def get_or_create_expectation_suite(
 
 
 def get_default_expectation_suite_name(
-    data_asset_name: str,
+    data_asset_name: Union[str, None],
```

Comment:
@Kilo59 Would `data_asset_name: Optional[str]` not be appropriate shorter version?  Thanks.

Reply:
See above.